app/assets/scripts/views/project.js

```diff
@@ -189,31 +190,50 @@ var Project = React.createClass({
                   </ul>
                 </div>
 
-                <div className='overview-item'>
-                  <h2 className='overview-item__title heading-alt'>Project Link</h2>
-                  <ul className='link-list'>
-                    <li><a href='' className='link--primary'><span>Name of Source</span></a></li>
-                  </ul>
-                </div>
-
-                <div className='overview-item'>
-                  <h2 className='overview-item__title heading-alt'>Responsible Party</h2>
-                  <ul className='link-list'>
-                    <li><a href='' className='link--primary'><span>Name of Responsible Party</span></a></li>
-                  </ul>
-                </div>
+                {data.project_link && (
+                  <div className='overview-item'>
+                    <h2 className='overview-item__title heading-alt'>Project Link</h2>
+                    <ul className='link-list'>
+                      <li><a href={data.project_link} className='link--primary'><span>Link</span></a></li>
+                    </ul>
+                  </div>
+                )}
+
+                {data.responsible_ministry && data.responsible_ministry.toLowerCase() !== 'select a ministry' && (
+                  <div className='overview-item'>
+                    <h2 className='overview-item__title heading-alt'>Responsible Ministry</h2>
+                    <ul className='link-list'>
+                      <li><a href='' className='link--primary'><span>{data.responsible_ministry}</span></a></li>
+                    </ul>
+                  </div>
+                )}
+
+                {data.local_manager && data.local_manager.toLowerCase() !== 'select a party' && (
+                  <div className='overview-item'>
+                    <h2 className='overview-item__title heading-alt'>Local Manager</h2>
+                    <ul className='link-list'>
+                      <li><a href='' className='link--primary'><span>{data.local_manager}</span></a></li>
+                    </ul>
+                  </div>
+                )}
 
-                <div className='overview-item'>
-                  <h2 className='overview-item__title heading-alt'>Responsible Party</h2>
+                <div className='overview-item--alt'>
```

Comment:
Good question, not actually sure if we want to hide the entire thing or show a 'NA' or '--' or other sign to indicate there's no data. Leaving this up to @felskia.